src/library_sdl.js

```diff
@@ -1252,7 +1252,9 @@ var LibrarySDL = {
         if (state === null) return;
         // Check only if the timestamp has differed.
         // NOTE: Timestamp is not available in Firefox.
-        if (typeof state.timestamp !== 'number' || state.timestamp !== prevState.timestamp) {
+        // NOTE: Timestamp is currently not properly set for the GearVR controller
+        // on Samsung Internet: it is always zero.
```

Comment:
please indent "on" to match "Timestamp" - that way it's clear it's all a single note